xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/DXVAHD.h

```diff
@@ -104,6 +105,7 @@ class CProcessorHD : public ID3DResource
   D3D11_VIDEO_PROCESSOR_RATE_CONVERSION_CAPS m_rateCaps;
   D3D11_TEXTURE2D_DESC m_texDesc;
   PROCESSOR_VIEW_TYPE m_eViewType;
+  std::vector<AVPixelFormat> m_formats;
```

Comment:
In case of sw decoding and DXVA render method enabled with yuv420p10 video. DXVA may not support yuv420p10 format, actually it is possible to detect supported fornat after initing DXVA renderer
it's not a problem to drop it, but I think it will be a pain